src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -55,7 +55,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _settingsClone{ settings.Copy() }
     {
         InitializeComponent();
-
+        _updateBackgroundForMica();

```

Comment:
```suggestion
        _UpdateBackgroundForMica();
```
nit: all the other ones start capitalized